layouts/partials/header.html

```diff
@@ -141,9 +141,6 @@ <h2 class="hero-section__subtitle">{{ .subtitle }}</h2>
                     class="lni lni-book"></i><span>Documentation</span></a>
         </div>
     </div>
-    <div class="hero-section__separator">
```

Comment:
The separator is pretty IMO, we can use it for something else, e.g our latest blogpost, or something.
wdyt? Now the website (in terms of design) looks a bit less good

Reply:
Sounds like a good idea to me! I will add this.